### ISSUE TITLE
Boettcher's function

### ISSUE THREAD

**Issue by adammaj1** [fredrik-johansson/fungrim#23]
Hi,

Is it possible to add : 
https://en.wikipedia.org/wiki/B%C3%B6ttcher%27s_equation
https://reference.wolfram.com/language/ref/MandelbrotSetBoettcher.html

TIA

**Comment by fredrik-johansson** [comment]
Yes, of course. To be a bit more systematic about this, I picture a section on the front page on "Fractals" which could list topics on things such as the Weierstrass function, the Mandelbrot set, Julia sets...

In the topic for the Mandelbrot set, you could have things like the Mandelbrot set itself as an object, the Boettcher function, and other things that make sense to document.
